sdjournal/read.go

```diff
@@ -38,7 +38,10 @@ type JournalReaderConfig struct {
 	// the array is empty, entries will not be filtered.
 	Matches []Match
 
-	MaxEntryLength uint64 // the max allowable length for an entry (default is 64K)
+	// explicitly set MaxEntryLenght to 0, which means unlimited
+	DisableMaxEntryLength bool
+	// the max allowable length for an entry (default is 64K)
+	MaxEntryLength uint64
```

Comment:
@runcom How about just scrapping this bit of the config struct entirely, and requiring users to call `SetDataThreshold()` explicitly after they instantiate a `JournalReader`? That seems like the cleanest path forward, imo.